src/Fhir.Anonymizer.Stu3.Tool/Fhir.Anonymizer.Stu3.CommandLineTool.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Could we rename the folder name from "tool" to "tools"? It seems all other folders are in plural form.